docs/pages/blog/2023-toolpad-beta-announcement.md

```diff
@@ -0,0 +1,81 @@
+
+<img alt="Building an application on Toolpad " src="/static/blog/2023-toolpad-beta-announcement/toolpad.png" style="margin-bottom: 16px;" loading="lazy" width="2048" height="1536" />
+
+## Who is Toolpad for?
+
+Before the arrival of Toolpad, MUI primarily catered to the front-end needs of the development process. Our products were mostly useful for React engineers or designers. We saw many full-stack engineers use our advanced components for data-intensive apps. These highly customizable components are well-suited for complex apps, but not all apps require the same level of customization: for example, internal tools just need basic theming.
+
+Toolpad is for full-stack and back-end developers. They build, manage, and integrate data pipelines, server-side logic, databases, microservices, and APIs—and know the most about them. Sharing these endpoints and maintaining the internal documentation with the front-end teams is a hassle that lasts forever. Toolpad empowers you to take the next step and build the UI you need using our drag-and-drop editor.
+
+## What are the main features available today?
+
+### Use MUI Core and MUI X Pro components
+
+MUI Core provides all modern components; a chosen few are currently available inside Toolpad, and we're adding more all the time. MUI X Pro (Data Grid and Date Picker) components are also available for free in Toolpad.
+
+### Bring your own components
+
+Toolpad supports bringing your [external components](https://mui.com/toolpad/concepts/custom-components/) to the project. Your past effort should not go to waste—we don't believe in a walled garden approach. In the future, we'll also support importing from Storybook and GitHub.
+
+### Bring your own back end
+
+Directly integrate with your Node.js back end and have your data available on the page without writing any REST endpoints or fetch logic.
+
+### Write custom functions in your IDE
+
+Toolpad works with your code editor and lets you write [custom functions](https://mui.com/toolpad/concepts/connecting-to-data/#custom-functions). These functions:
+
+- run on the back end.
+- return the data that is made available on the page without requiring you to set up any endpoints or data-fetching logic.
+- support arguments can be bound directly to page state.
+- can be integrated not only with any node module, but also with your back-end code, such as Prisma models or database connections.
+
+### Own your code
+
+Toolpad runs fully locally. All configuration is stored in local files which you can version-control, edit, and deploy any way you want.
+
+## How is Toolpad different from similar products on the market?
+
+Toolpad follows an open-core approach, meaning that the base version is open source. You can build an unlimited number of apps, and the basic service will be free forever. Among similar products like Retool, Appsmith, Budibase, Airplane, and Interval, only some are open source.
```

Comment:
Feels a bit like a weak point. "We're open-source just like others". Besides, "open-core", "open-source" and "free" are terms with different meanings I feel like all it will do is annoy the people who truly care about free software ("free" doesn't always mean "as in beer"). I'd suggest dropping this paragraph.

Reply:
I can see the value in naming our competitors here to encourage people to associate us with them. We want readers to see Toolpad as a viable alternative to these products, and it's good to point out what makes us unique. But I do agree that what's being pointed out here is a rather weak point.